src/esp/gfx/RenderCamera.h

```diff
@@ -43,6 +43,11 @@ class RenderCamera : public MagnumCamera {
   typedef Corrade::Containers::EnumSet<Flag> Flags;
   CORRADE_ENUMSET_FRIEND_OPERATORS(Flags)
 
+  typedef std::vector<
+      std::pair<std::reference_wrapper<Magnum::SceneGraph::Drawable3D>,
+                Magnum::Matrix4>>
+      DrawableTransforms;
```

Comment:
Perhaps apply this typedef to [CullingTest.cpp](https://github.com/facebookresearch/habitat-sim/blob/ae2faa8f6d726dd73c4fa5a7b1880b61b2bfab24/src/tests/CullingTest.cpp#L180) as well?